src/notification/notification.service.ts

```diff
@@ -364,4 +365,39 @@ export class NotificationService {
             ])
             .delete();
     }
+
+    async createMentionNotification(mention: Mention) {
+        const user = await this.db('users')
+            .where('account_id', mention.accountId)
+            .select('id')
+            .first();
+
+        if (!user) {
+            // If the mention is for an account that no longer exists or is external,
+            // don't create a notification
+            return;
+        }
+
+        const post = await this.db('posts')
```

Comment:
We can skip this db call if we already pass the post to `createMentionNotification`

Reply:
removed in 731d4d4d24dd984e00f9b9bead0a6c1ac8b463d8